package.json

```diff
@@ -162,17 +148,9 @@
         {
           "provider": "generic",
           "url": "https://updates.signal.org/desktop"
-        },
-        {
-          "provider": "s3",
-          "region": "us-east-1",
-          "bucket": "updates.signal.org",
-          "path": "desktop",
-          "acl": "public-read"
         }
       ],
       "target": [
-        "dmg",
```

Comment:
I’ve always wondered: What are the pros and cons for DMG?

Reply:
It's more of a natural install mechanism for OSX, honestly. You get a nice little 'Drag this icon to this thing' interface. But all of our install links go to the zip right now. So ZIP it is, at least for now.